specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/examples/alertRuleTemplates/GetAlertRuleTemplates.json

```diff
@@ -33,9 +33,7 @@
               "requiredDataConnectors": [
                 {
                   "connectorId": "AWS",
-                  "dataTypes": {
-                    "AWSCloudTrail": "Exist"
```

Comment:
>"AWSCloudTrail": "Exist" [](start = 20, length = 24)

This appears to be a breaking change. Existing clients will not be able to deserialize the new format.

Reply:
Yes, it is. We are changing our **preview version of the API**..

Reply:
Having the "preview" suffix in the API version is not a license to break existing clients. Typically these types of changes are introduced in a new API version and the old API version maintains the old serialization. 

If this change were allowed, how many users would be broken by the change? Are they internal or external? Are there any SDKs that were published?

Reply:
Hi, @majastrz 
As I wrote you in mail, we have only **few** users (~3 subscriptions) using the API from outside Azure Portal. The Azure Portal was adjusted to work both with old API model and new API model, and is working properly with both. Our PMs also have approved this change.
Practically, this API change was already arrived and is running in Production. 

This swagger change will align our swagger to reflect our current behaviour of API and will allow users to update their clients.
Please help us complete this PR

Reply:
Got it. Since you're making the swagger reflect API behavior, it's fine.

---
In reply to: [375109249](https://github.com/Azure/azure-rest-api-specs/pull/8130#discussion_r375109249) [](ancestors = 375109249)